src/main/java/it/gov/pagopa/payhub/auth/service/user/UserServiceImpl.java

```diff
@@ -51,14 +55,21 @@ public UserInfo getUserInfo(String accessToken) {
         IamUserInfoDTO userInfo = tokenStoreService.load(accessToken);
         if (userInfo == null) {
             throw new InvalidAccessTokenException("AccessToken not found");
-        } else {
-            return userInfoMapper.apply(userInfo);
         }
+
+        UserInfo result = userInfoMapper.apply(userInfo);
+
+        result.setCanManageUsers(!organizationAccessMode);
+
+        log.debug("User info retrieved successfully with brokerId: {}",
+                result.getBrokerId() != null ? result.getBrokerId() : "N/A");
```

Comment:
this ternary operator could be removed
```suggestion
                result.getBrokerId());
```